src/ngraph/ngraph_emitter.cc

```diff
@@ -695,6 +695,35 @@ void Emitter::CreateBinaryOps() {
     return cast_result(CreateAutoBroadcast<ngraph::op::LessEq>(node),
                        getType(node->dtype_));
   };
+  ngraph_op_funcs_["broadcast_to"] =
+      [this](const NodePtr& node) -> NgraphNodePtr {
+    auto input = op_map_[node->inputs_[0]];
+    const auto& input_shape = input->get_shape();
+    ngraph::Shape output_shape(
+        get_default(node, "shape", std::vector<size_t>{}));
+    ngraph::AxisSet broadcast_axis;
+    ngraph::Shape proxy_shape;
+    assert(ngraph::shape_size(input_shape) == ngraph::shape_size(output_shape));
+    // ngraph::op::broadcast does not allow in place broadcast (must add
+    // new axis), so we reshape the input and eliminate axis with dim 1,
```

Comment:
IMO the word "dim[ension]" is a little ambiguous here.  Would it be clearer to instead write "... with *span* 1" or "... with *length* 1"?